workspace.json

```diff
@@ -5091,13 +5091,10 @@
             "passWithNoTests": true
           }
         },
-        "schemas/openapi-generator": {
+        "schemas/external-openapi-generator": {
```

Comment:
Why?

Reply:
To [generate code for external services](https://github.com/island-is/island.is/pull/2528/files#diff-70961dbd723236df99e3984296be3e434e7600693bc03044c6f7d817e1bbe021R21) before generating code for libs/apps that depend on them.